test/utils/Strings.test.js

```diff
@@ -12,15 +12,15 @@ describe('Strings', function () {
 
   describe('from uint256', function () {
     it('converts 0', async function () {
-      expect(await this.strings.fromUint256(0)).to.equal('0');
+      expect(await this.strings.toString(0)).to.equal('0');
```

Comment:
I think this is clashing with the JavaScript `toString` method, which causes the tests to fail. We should use the `contraxt.methods['toString()']` syntax.